### DISCUSSION TITLE
Development environment setup issue

### DISCUSSION THREAD

**Discussion by Adebesin-Cell** [aspen-cloud/triplit#204] [General]
I'm trying to setup the dev environment locally, and keep getting this while trying to run ``yarn install``, is there anything to follow perhaps steps to get started for contribution?

<img width="1006" alt="Screenshot 2024-09-10 at 4 44 23 PM" src="https://github.com/user-attachments/assets/21ee176e-f448-4774-a88b-e8ce84ccc849">

**Comment by matlin** [comment]
Which version of Node are you on? I think there are issues with version 22 so you might want to use Node 20. If you aren't already, I recommend using [nvm](https://github.com/nvm-sh/nvm)

**Reply by Adebesin-Cell** [reply]
Ha, yeah, I'm using 22. Alright, I'll use that, thank you 👏

**Reply by Adebesin-Cell** [reply]
Yeah, it worked! Thanks @matlin

**Comment by Adebesin-Cell** [comment]
Hey @matlin, quick one, I wanted to start contributing, a bit lost in space as to where to start from though pin pointed to start working on some refactor/some code update to get familiar with the project, if you're willing to guide, I could reach out to you on discord.
